cypress/integration/advanced-functionalities/server/Advanced_functionalities.spec.js

```diff
@@ -0,0 +1,106 @@
+/// <reference types="cypress" />
+const baseURL = Cypress.env('baseURL')
+const newEmail = Cypress.env('user').newEmail
+const newPassword = Cypress.env('user').newPassword
+
+describe('Test server advanced functionalities ', () => {
+  beforeEach(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ baseURL })
+      cy.get('[data-test-id="input-email"]').type(newEmail)
```

Comment:
can't you pass all of these params to the login function as we do in other files?